modules/virt-editing-vmis-web.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/cnv-manage-vmis.adoc
+
+[id="virt-editing-vmis-web_{context}"]
+= Editing a standalone virtual machine instance using the web console
+
+You can edit text and annotations for a standalone virtual machine instance using the web console. Other items displayed in the *Overview* page for a standalone virtual machine instance are not editable.
+
+.Procedure
+
+. Click *Workloads -> Virtual Machines*  from the side menu.
+
+. Select a virtual machine instance to open the *Virtual machine instance overview* screen.
+
```

Comment:
It needs a step to navigate to `details` tab.

Reply:
Added the step. Please review.

Reply:
`labels` don't have editor pencil. It's "annotations" can be edited directly in the `Details` page.

Reply:
Updated the content. Please review.

Reply:
Labels don't have icon pencil, so it cannot be editing in the details page. It can be edited by "Actions" -> "Edit Labels". Suggest remove step 3 and 4.

Reply:
To ensure that I was accurately documenting the UI, I went and looked at the 2.3 UI and to extrapolate how the VMI Overview screen is laid out. I updated this particular procedure, and I felt that mentioning how to edit the labels was important. Therefore, please review the procedure now. I have mentioned about how to select Actions -> Edit Labels in the VMI Overview screen to show the user how labels are edited. Thanks! Please review and advise.

Reply:
This changes looks good, glad you take a look at UI.

Reply:
Thanks for reviewing! Verified with @matthewcarleton during our review of the user interface session that labels cannot be edited in the VMI Details page. He noted that there is a defect open and this would be addressed in the future.